package.json

```diff
@@ -14,73 +14,71 @@
     "lint": "eslint './{src,example}/**/*.{ts,tsx}' && tsc --noEmit",
     "test": "jest",
     "build": "yarn lint && yarn test && rm -rf dist && yarn build:compile",
-    "build:compile": "microbundle --format cjs,umd",
+    "build:compile": "microbundle --format cjs,umd --globals @apollo/react-common=apolloReactCommon,apollo-link=apolloLink",
```

Comment:
To be in sync with [the config from react-apollo](https://github.com/apollographql/react-apollo/blob/4a9f9b6bb15a2ec6dab95170757e00e2652f5d4b/config/rollup.config.js#L11-L33).

This allows to load the library unbundled as a script tag.